admin_2017/app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
`.html.erb` vs. `.erb`

Reply:
Oh, I remember, I believe the rails generators by default supply a application.html.erb; this generator tries to overwrite it.